### PR TITLE
DOC: clean up some doc-build warnings

### PR DESCRIPTION
**Pull request by jreback**

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -4945,11 +4947,12 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
         Returns
         -------
         quantiles : Series or DataFrame
-            If ``q`` is an array, a DataFrame will be returned where the
-            index is ``q``, the columns are the columns of self, and the
-            values are the quantiles.
-            If ``q`` is a float, a Series will be returned where the
-            index is the columns of self and the values are the quantiles.
+
+            - If ``q`` is an array, a DataFrame will be returned where the
```
Comment (jreback) [inline pandas/core/frame.py]: so @jorisvandenbossche all of those natsy warnings just needed a blank line before AND after a list when embedded in a doc-string. very confusing message

**Comment by jreback** [comment]
only big ones left are these

```
/Users/jreback/pandas/doc/source/generated/pandas.Series.str.partition.rst:6: WARNING: don't know which module to import for autodocumenting u'Series.str.partition' (try placing a "module" or "currentmodule" directive in the document, or giving an explicit module name)
/Users/jreback/pandas/doc/source/generated/pandas.Series.str.repeat.rst:6: WARNING: don't know which module to import for autodocumenting u'Series.str.repeat' (try placing a "module" or "currentmodule" directive in the document, or giving an explicit module name)
```

File: pandas/core/frame.py (hunk 1 above)
Comment (jorisvandenbossche) [inline pandas/core/frame.py]: Yes, indeed, that's the rule for rst lists and also apply in docstrings, *unless* it's the only thing in the explanation of a parameter (no text before or after in the entry), then no blank lines are needed (eg in this case it is normally not needed)

File: pandas/core/generic.py [hunk 2]
```diff
@@ -2041,11 +2041,13 @@ def sort_index(self, axis=0, level=None, ascending=True, inplace=False,
             method to use for filling holes in reindexed DataFrame.
             Please note: this is only  applicable to DataFrames/Series with a
             monotonically increasing/decreasing index.
+
               * default: don't fill gaps
               * pad / ffill: propagate last valid observation forward to next
                 valid
               * backfill / bfill: use next valid observation to fill gap
               * nearest: use nearest valid observations to fill gap
+
```
Comment (jorisvandenbossche) [inline pandas/core/generic.py]: In the list above, the extra indentation for the list is also not needed

File: pandas/tseries/tools.py [hunk 3]
```diff
@@ -214,14 +218,18 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         Return UTC DatetimeIndex if True (converting any tz-aware
         datetime.datetime objects as well).
     box : boolean, default True
+
         - If True returns a DatetimeIndex
         - If False returns ndarray of values.
+
```
Comment (jorisvandenbossche) [inline pandas/tseries/tools.py]: For these, the extra blank space should normally not be needed (as I explained above). Did you get warnings for these as well?

**Comment by jorisvandenbossche** [comment]
On the `groupby.nth`, I also already looked at before, but didn't spot the error either

File: pandas/core/frame.py (hunk 1 above)
Comment (jreback) [inline pandas/core/frame.py]: ok I fixed, but that seems a confusing rule. better to just put blanks around lists in doc-strings always, no?  (checking if taking these out still has warnings). I was trying things so didn't always take out the added whitespace.

**Comment by jreback** [comment]
merged via https://github.com/pydata/pandas/commit/fcab956d41ce74a927f30b3723f0fdfde7df75b6

**Pull request closed without merging**